nodebuilder/state/core.go

```diff
@@ -20,6 +20,11 @@ func coreAccessor(
 	sync *sync.Syncer[*header.ExtendedHeader],
 	fraudServ libfraud.Service,
 ) (*state.CoreAccessor, *modfraud.ServiceBreaker[*state.CoreAccessor]) {
+	if !corecfg.EndpointConfigured() {
+		log.Info("No core endpoint provided, running node without state access")
```

Comment:
I think we should start hardcodin our Core node bootstrappers so that everyone has state access by default. Otherwise, we are not really usable with just a few CLIs.

Reply:
https://github.com/celestiaorg/celestia-node/issues/2579